File: openapi-generator/dart/lib/model/dvir_base_trailer_defects.dart
part of openapi.api;

class DvirBaseTrailerDefects {
  /* The comment describing the type of DVIR defect */
  String comment = null;
  /* The type of DVIR defect */
  String defectType = null;
  DvirBaseTrailerDefects();

  @override
  String toString() {
    return 'DvirBaseTrailerDefects[comment=$comment, defectType=$defectType, ]';
  }

  DvirBaseTrailerDefects.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    if (json['comment'] == null) {
      comment = null;
    } else {
          comment = json['comment'];
    }
    if (json['defectType'] == null) {
      defectType = null;
    } else {
          defectType = json['defectType'];
    }
  }

  Map<String, dynamic> toJson() {
    return {
      'comment': comment,
      'defectType': defectType
    };
  }

  static List<DvirBaseTrailerDefects> listFromJson(List<dynamic> json) {
    return json == null ? new List<DvirBaseTrailerDefects>() : json.map((value) => new DvirBaseTrailerDefects.fromJson(value)).toList();
  }

  static Map<String, DvirBaseTrailerDefects> mapFromJson(Map<String, dynamic> json) {
    var map = new Map<String, DvirBaseTrailerDefects>();
    if (json != null && json.length > 0) {
      json.forEach((String key, dynamic value) => map[key] = new DvirBaseTrailerDefects.fromJson(value));
    }
    return map;
  }
}

File: openapi-generator/csharp-dotnet2/src/main/CsharpDotNet2/Org/OpenAPITools/Model/DvirBaseTrailerDefects.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace Org.OpenAPITools.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class DvirBaseTrailerDefects {
    /// <summary>
    /// The comment describing the type of DVIR defect
    /// </summary>
    /// <value>The comment describing the type of DVIR defect</value>
    [DataMember(Name="comment", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "comment")]
    public string Comment { get; set; }

    /// <summary>
    /// The type of DVIR defect
    /// </summary>
    /// <value>The type of DVIR defect</value>
    [DataMember(Name="defectType", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "defectType")]
    public string DefectType { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class DvirBaseTrailerDefects {\n");
      sb.Append("  Comment: ").Append(Comment).Append("\n");
      sb.Append("  DefectType: ").Append(DefectType).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
